bigtop_toolchain/bin/puppetize.sh

```diff
@@ -26,7 +26,7 @@ case ${ID}-${VERSION_ID} in
       dnf -y check-update
       dnf -y install hostname findutils curl sudo unzip wget puppet puppetlabs-stdlib
         ;;
-    ubuntu-16.04)
+    ubuntu-1[6,8].04)
```

Comment:
Would be better to create another entry for 18.04 as there are different packages to be installed.